client/blocks/comment-detail/index.jsx

```diff
@@ -80,18 +81,40 @@ export class CommentDetail extends Component {
 
 	edit = () => noop;
 
+	getCommentObject = () => ( {
+		commentId: this.props.commentId,
+		isLiked: this.props.commentIsLiked,
+		postId: this.props.postId,
+		status: this.props.commentStatus,
+	} );
```

Comment:
it would be much better here I think to pass in the props at the call site and let this be a normal function outside of the React component…

```js
const getCommentObject = ( { commentId, commentIsLiked, commentStatus, postId } ) => ( {
	commentId,
	commentStatus,
	isLiked: commentIsLiked,
	postId,
} )
```